scripts/get-alphanet-relay-image.sh

```diff
@@ -0,0 +1,17 @@
+#/bin/sh
+
+POLKADOT_COMMIT=$(egrep -o '/polkadot.*#([^\"]*)' Cargo.lock | head -1 | sed 's/.*#//' |  cut -c1-8)
+DOCKER_TAG="purestake/moonbase-relay-testnet:sha-$POLKADOT_COMMIT"
+
+# Build relay binary if needed
+POLKADOT_EXISTS=docker manifest inspect $DOCKER_TAG > /dev/null && "true" || "false"
+if [[ "$POLKADOT_EXISTS" == "false" ]]; then
+  # $POLKADOT_COMMIT is used to build the relay image
+  ./scripts/build-alphanet-relay-image.sh
+  docker push $DOCKER_TAG
```

Comment:
We should not push from the local, only from the CI

Reply:
fixed